API_Auto/src/test/java/postAllProduct/AllProduct.java

```diff
@@ -26,21 +26,18 @@ public AllProduct() {
 
     @Test(priority = 1)
     public void checkStatusCode() {
-        Response response = RestAssured.post(url)
-                .then().extract().response();
+        Response response = RestAssured.post(url).then().extract().response();
         Assert.assertEquals(response.getStatusCode(), 200);
-        System.out.println("The response code is : " + response.getStatusCode());
         log.info("The response code is : " + response.getStatusCode());
     }
 
     @Test(priority = 2)
     public void checkResponse() {
-        Response response = RestAssured.post(url)
-                .then().extract().response();
+        Response response = RestAssured.post(url).then().extract().response();
         JsonPath jsonResponse = new JsonPath(response.asString());
         ResponseBody body = response.getBody(); //use to print response body
         log.info("Response Body is: " + body.asString());
-        log.info("The response message is : " + jsonResponse.get("message"));
-        log.info("The response code is : " + jsonResponse.get("responseCode"));
+        Assert.assertEquals(response.getStatusCode(), 200);
```

Comment:
Resolve the conflicts and remove the unnecessary logs

Reply:
done